### ISSUE TITLE
Why do we multiply by 0.5 and adding 0.5 when calculatin PSNR, SSIM and MS-SSIM ?

### ISSUE THREAD

**Issue by JauB1981** [DeokyunKim/Progressive-Face-Super-Resolution#6]
Hi
Why do we multiply by 0.5 and adding 0.5 when calculatin PSNR, SSIM and MS-SSIM ?
i.e  here :
` mse_loss = MSE_Loss**(0.5***predicted_image+**0.5**, **0.5***target_image+**0.5**)    
 _ssim = ssim(**0.5***predicted_image+**0.5**, **0.5***target_image+**0.5**)

ms_ssim = msssim(**0.5***predicted_image+**0.5**, **0.5***target_image+**0.5**)`

And if I want to compare for example PSNR for Progressive Face SR and PSNR of FSRNet, should i do the same operation by multiplying by 0.5 and adding 0.5 or it depends on the Network ?
Thanks.

**Comment by DeokyunKim** [comment]
Becausw the dataset is normalized by ((0.5, 0.5, 0.5), (0.5, 0.5, 0.5)). The uploaded model is trained with normalized dataset. Therefore, we multiply and add 0.5

**Comment by JauB1981** [comment]
OK thanks.
I got it ;)
